rest/client.go

```diff
@@ -43,14 +45,18 @@ func NewClient(opts ...rest.Option) MeasurementsClient {
 	return &client{Client: restClient}
 }
 
-func (c *client) GetNodeDataRecent(ctx context.Context, nodeID uuid.UUID, contentType []string) (models.ModelNodeDataResponse, error) {
+func (c *client) GetNodeDataRecent(ctx context.Context, nodeID uuid.UUID, contentTypes []string) (models.ModelNodeDataResponse, error) {
 	request := rest.Get("nodes/{nodeID}/node-data/recent{?content_type*}").
 		Assign("nodeID", nodeID.String()).
-		Assign("content_type", contentType).
+		Assign("content_type", workaround.FormatContentTypes(contentTypes)).
 		SetHeader("Accept", "application/json")
 
 	var response models.ModelNodeDataResponse
-	if err := c.DoAndUnmarshal(ctx, request, &response); err != nil {
+
+	err := c.DoAndUnmarshal(ctx, request, &response)
+	if errors.Is(err, rest.ErrNotFound) {
+		return models.ModelNodeDataResponse{}, ErrNotFound
+	} else if err != nil {
```

Comment:
If you make the `ErrNotFound` a simple re-export of the `rest.ErrNotFound` this will not be needed.

Reply:
You mean `var ErrNotFound = rest.ErrNotFound`

Reply:
Yes, I see :) That's quite nice